cluster-scope/base/operators.coreos.com/subscriptions/koku-metrics-operator/subscription.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: koku-metrics-operator-sub
```

Comment:
you should just name it `koku-metrics-operator`

Reply:
Updated the name.